src/AddTodoForm.jsx

```diff
@@ -1,21 +1,43 @@
+import Button from "../public/components/Button";
 import PropTypes from "prop-types";
-Button.propTypes = {
-  children: PropTypes.node.isRequired,
+import { useState } from "react";
+
+AddTodoForm.propTypes = {
+  onAddTodo: PropTypes.func.isRequired,
 };
 
-export default function AddTodoForm() {
+export default function AddTodoForm({ onAddTodo }) {
+  const [todoTitle, setTodoTitle] = useState("");
+
+  function handleAddTodo(e) {
+    e.preventDefault();
+
+    const newToDoList = {
+      id: Date.now(),
+      title: todoTitle,
```

Comment:
The value of the title should be retrieved from the event instead of being stored in a state.